ocaml/otherlibs/stdlib_alpha/capsule.mli

```diff
@@ -40,25 +40,68 @@
     this interface we will often use ['k] to refer to the capsule associated
     with that brand. *)
 
-(** Passwords represent access to a capsule. *)
+(** Converters can wrap and unwrap [Data.t] values from the current
+    capsule. *)
+module Converter : sig
+
+    (* CR layouts v5: this should have layout [void], but
+       [void] can't be used for function argument and return types yet. *)
+    type 'k t : value mod global portable many unique
+    (** ['k t] is the type of "converters" that can wrap and unwrap
+        [Data.t] values. An [uncontended] ['k t] indicates that ['k] is the
```

Comment:
Does that mean `assert_current : 'a Converter.t -> 'b Converter.t -> ('a, 'b) Type.eq` is sound?

Reply:
Yeah, I guess it does. Maybe we should include it?

Reply:
I went ahead and added it